applications/TrilinosApplication/trilinos_space.h

```diff
@@ -115,17 +115,17 @@ class TrilinosSpace
     ///@name Operations
     ///@{
     //
-    static MatrixPointerType CreateEmptyMatrixPointer()
+    static MatrixPointerType CreateEmptyMatrixPointer(const DataCommunicator& rComm)
     {
         return MatrixPointerType(nullptr);
     }
 
-    static VectorPointerType CreateEmptyVectorPointer()
+    static VectorPointerType CreateEmptyVectorPointer(const DataCommunicator& rComm)
     {
         return VectorPointerType(nullptr);
     }
 
-    static MatrixPointerType CreateEmptyMatrixPointer(Epetra_MpiComm& Comm)
+    static MatrixPointerType CreateEmptyMatrixPointer(Epetra_MpiComm& Comm) //TODO: unify with the previous ones
```

Comment:
Comment should be in a Doxygen block?

Reply:
no, this is "internal suggestion" should not be in docs